website/templates/log_templates.mako

```diff
@@ -191,6 +191,19 @@ invisible on
   {{/ifnot}}
 </script>
 
+<script type="text/html" id="addon_file_renamed">
+  {{#if params.source.materialized.endsWith('/')}}
```

Comment:
@revanthkolli That will be awesome. And also we decided to remove the ending slash of folder in logs, if you update your code with develop, https://github.com/CenterForOpenScience/osf.io/blob/develop/website/static/js/logFeed.js#L87  will help you to remove the ending slash showing up in logs.